src/test/java/org/reaktivity/nukleus/tcp/internal/streams/rfc793/ServerIOExceptionFromWriteIT.java

```diff
@@ -90,7 +92,32 @@ public void shouldResetWhenImmediateWriteThrowsIOException() throws Exception
     @Test
     @Specification({
         "${route}/server/controller",
-        "${server}/server.sent.data.received.reset.and.abort/server"
+        "${server}/server.sent.data.received.abort.and.reset/server"
```

Comment:
This method should be removed for the same reasons I explained in the corresponding client test.

Reply:
Done.